schema/workflow.json

```diff
@@ -556,7 +557,26 @@
       },
       "additionalProperties": false,
       "required": [
-        "produceEventRef"
+        "event", "data"
+      ]
+    },
+    "subscribe": {
+      "type": "object",
+      "description": "Subscribe to an event channel",
+      "properties": {
+        "event": {
```

Comment:
Could you move the `event` definition out of both `subscribe` and `publish` and instead reference it?
It removes a duplicate and is therefore easier to maintain.

Reply:
I do not understand, right now event is just an string with the regular expression we are using for names (that is duplicated along the whole spec) , whats the benefit of replacing?
```
"event": {
          "type": "string",
          "description": "Reference to the unique name of a 'published' event definition",
          "pattern": "^[a-z0-9](-?[a-z0-9])*$"
        }
```
by
```
"event": {
    "type": "#/definitions/eventRef" 
}
```
I would see the benefit, but I think is matter of a clean up PR, to put all names under a new definition "nameWithReg"